apple/macbook-air/7/default.nix

```diff
@@ -0,0 +1,9 @@
+{ config, lib, pkgs, ... }:
+
+{
+  imports = [
+    ../.
+  ];
+
+  boot.extraModulePackages = lib.mkDefault [ config.boot.kernelPackages.broadcom_sta ];
```

Comment:
Does nixos-generate-config not also include this already?

Reply:
i'm not sure, the wifi didn't work initially, but i was able to get it working later, the macbook is currenly not booting, but if i get it fixed i'll let you know

Reply:
Afaik it should be in your hardware-configuration.nix. At least this is what I remember reading the perl code.

Reply:
ok, i removed it